repos/system_upgrade/el7toel8/libraries/rhsm.py

```diff
@@ -69,29 +70,37 @@ def _handle_rhsm_exceptions(hint=None):
             }
         )
     except CalledProcessError as e:
+        _def_hint = (
+            'Please ensure you have a valid RHEL subscription and your network is up.'
+            ' If you are using proxy for Red Hat subscription-manager, please ensure'
```

Comment:
```suggestion
            ' If you are using proxy for Red Hat subscription-manager, please make sure'
```

Reply:
done

Reply:
'Ensure' is pretty ok, the docs team has used it 7 times in the upgrade doc: https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/upgrading_to_rhel_8/index
But nevermind.